modules/installation-bare-metal-config-yaml.adoc

```diff
@@ -7,9 +7,23 @@
 ifeval::["{context}" == "installing-restricted-networks-bare-metal"]
 :restricted:
 endif::[]
+ifeval::["{context}" == "installing-ibm-z"]
```

Comment:
Is there a reason that IBM Z and power are using bare metal's config instead of having their own file (like the other cloud providers do)?

Reply:
To make use of modularity.
Plus IBM Z was already using the bare metal's config when I started work, so I went with it. Then, when creating the IBM Power one, it was very similar to the IBM Z one, so I did it the same way.
I think the real (small) problem here is the naming of this thrice-used file. Do you agree that I should rename it into `installation-config-yaml.adoc`? To remove the implication that it's supposed to be about bare metal and the other two are living off it.

Reply:
Fair enough. The name of the file is what caused me to ask. Though I don't know whether it's the right thing to do, to change the file name, since it is still only for a subset of the install options. Up to you, but I'd say it's not critical.

I would at least add the power assemblies that it's newly included for to the list of assemblies at the top of the module.

Reply:
Hmm, you are right. I couldn't come up with a reasonable name which would indicate that it's just for a subset. Leaving it as it is, might address in a future release.